ui/ui-components/components/layouts/Main.tsx

```diff
@@ -116,7 +116,7 @@ export default function Main(props) {
               bottom: 0,
               right: 0,
               padding: 20,
-              zIndex: 99,
+              zIndex: 2000,
```

Comment:
What was higher than 99z?

Reply:
The Bootstrap modal zindex is set in the 1000s. So when a toast error comes up while the modal is open, it appears behind the semi-transparent bg.